src/harness/testcases/WINNF_FT_S_SCS_testcase.py

```diff
@@ -185,6 +185,31 @@ def test_WINNF_FT_S_SCS_10(self,config_filename):
       # Check Registration Response
       self.assertEqual(response[0]['response']['responseCode'], 104)
 
+  def generate_SCS_11_default_config(self, filename):
+    """Generates the WinnForum configuration for SCS_11. """
+    # Create the configuration for blacklisted client cert/key path
+
+    config = {
```

Comment:
The existing code (e.g. line 218) uses a 4-space indent after `{`.  Please maintain that.